.clang-tidy

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
What's the difference between this and what we used in cpp-best-practices org?

Reply:
the extension and the comments, was my fault